doc/source/whatsnew/v0.25.0.rst

```diff
@@ -176,6 +176,8 @@ Performance Improvements
   int8/int16/int32 and the searched key is within the integer bounds for the dtype (:issue:`22034`)
 - Improved performance of :meth:`pandas.core.groupby.GroupBy.quantile` (:issue:`20405`)
 - Improved performance of :meth:`read_csv` by faster tokenizing and faster parsing of small float numbers (:issue:`25784`)
+- Improved performance of :meth:`read_csv` by cythonizing _concat_date_cols without conversion for integer/float zero
```

Comment:
pls make this more user accessible; you are referencing internal functions which no-knows about

Reply:
I tried to write better.